src/main/java/org/embulk/input/aurora_mysql_extract_files/DefaultRetryable.java

```diff
@@ -158,19 +157,42 @@ public <X extends Throwable> T executeWithCheckedException(RetryExecutor executo
                 return this.call();
             }
             catch (Exception e) {
-                Throwables.propagate(e);
+                throw propagate(e);
             }
         }
 
         try {
             return executor.runInterruptible(this);
         }
         catch (RetryGiveupException e) {
-            Throwables.propagateIfInstanceOf(e.getCause(), propagateAsIsException);
-            throw Throwables.propagate(e.getCause());
+            propagateIfInstanceOf(e.getCause(), propagateAsIsException);
+            throw propagate(e.getCause());
         }
         catch (InterruptedException e) {
-            throw Throwables.propagate(e);
+            throw propagate(e);
+        }
+    }
+
+    @Deprecated
+    // https://github.com/google/guava/blob/v18.0/guava/src/com/google/common/base/Preconditions.java#L208-L213
+    // https://github.com/google/guava/blob/v18.0/guava/src/com/google/common/base/Throwables.java#L158-L161
+    // https://github.com/google/guava/blob/v18.0/guava/src/com/google/common/base/Throwables.java#L82-L85
+    private static RuntimeException propagate(Throwable throwable)
```

Comment:
Ported from Guava v18.0.
However, it has already been removed in the latest Guava.
https://github.com/google/guava/wiki/Why-we-deprecated-Throwables.propagate